client/ayon_core/hosts/max/api/lib.py

```diff
@@ -356,6 +351,26 @@ def get_max_version():
     return max_info[7]
 
 
+def get_current_task_entity():
+    """Function to get current task entity data
+
+    Returns:
+        dict: data of task entity
+    """
+    project_name = get_current_project_name()
+    folder_path = get_current_folder_path()
+    task_name = get_current_task_name()
+    folder_entity = ayon_api.get_folder_by_path(
+        project_name, folder_path, fields={"id"})
+    task_entity = ayon_api.get_task_by_name(
+        project_name,
+        folder_entity["id"],
+        task_name,
+        fields={"attrib"}
+    )
+    return task_entity
+
+
```

Comment:
@moonyuet this can now be removed with this PR merged: https://github.com/ynput/ayon-core/pull/353

```suggestion
```